src/main/java/org/example/ebanking/DBConnector.java

```diff
@@ -13,7 +11,7 @@ private Connection openConnection() {
         try {
             Class.forName("com.mysql.cj.jdbc.Driver");
             con = DriverManager.getConnection(CONNECT_STRING);
-            con.setAutoCommit(false);
+            con.setAutoCommit(true);
```

Comment:
Die DB commits sollten manuell mittels `con.commit();` durchgeführt werden. 

```suggestion
            con.setAutoCommit(false);
```